src/core/Akka/Actor/ActorSelection.cs

```diff
@@ -22,31 +23,32 @@ namespace Akka.Actor
     public class ActorSelection : ICanTell
     {
         /// <summary>
-        ///     Gets the anchor.
+        /// Gets the anchor.
         /// </summary>
         /// <value>The anchor.</value>
         public IActorRef Anchor { get; private set; }
 
         /// <summary>
-        ///     Gets or sets the elements.
+        /// Gets the elements.
         /// </summary>
         /// <value>The elements.</value>
-        public SelectionPathElement[] Path { get; set; }
+        public SelectionPathElement[] Path { get; private set; }
```

Comment:
Need to keep `Anchor` and `Path` with private setter otherwise a remote selection wouldn't work anymore, I assume it's the serialization issue with Newtonsoft < 8 and getter only properties